### ISSUE TITLE
Warnings from ESLint

### ISSUE THREAD

**Issue by blimmer** [codeclimate/codeclimate-eslint#142]
It looks like codeclimate marks new warnings in results? Is that expected? Or should it ignore those warnings? Just curious.

https://codeclimate.com/github/blimmer/test-codeclimate-eslint/issues

**Comment by dblandin** [comment]
Hey @blimmer,

That's a good question. Code Climate doesn't yet have the functionality to ignore issues based on severity, but it looks like you can control whether warnings are emitted in ESLint with the `--quiet` flag:

> Handling warnings
> `--quiet`

>This option allows you to disable reporting on warnings. If you enable this option, only errors are  reported by ESLint.

> Example:
> `eslint --quiet file.js`

source: http://eslint.org/docs/user-guide/command-line-interface#handling-warnings

I could see this option managed via the `.codeclimate.yml` config file. Enabling it would prompt the engine to append the `--quiet` flag.

```yaml
---
engines:
  eslint:
    enabled: true
    channel: eslint-3
    config:
      ignore_warnings: true
```

A downside is that you wouldn't see these warnings anywhere in the codeclimate.com interface, since they would cease to be emitted by the engine. Might be a good workaround for now though.

**Comment by dblandin** [comment]
This was implemented in #144. Going to go ahead and close this issue! Thanks!
